bin/citrea/src/main.rs

```diff
@@ -53,9 +57,18 @@ enum SupportedDaLayer {
 
 #[tokio::main]
 async fn main() -> Result<(), anyhow::Error> {
-    initialize_logging(tracing::Level::INFO);
-
     let args = Args::parse();
+
+    let logging_level = match args.verbose {
+        0 => tracing::Level::ERROR,
+        1 => tracing::Level::WARN,
+        2 => tracing::Level::INFO,
+        3 => tracing::Level::DEBUG,
+        4 => tracing::Level::TRACE,
+        _ => tracing::Level::INFO,
+    };
```

Comment:
if default for v is 2

how do i set it for 0 ?

Reply:
Added quiet arg